src/App/VPSScan/Types.hs

```diff
@@ -10,38 +10,33 @@ module App.VPSScan.Types
 , HTTPRequestFailed(..)
 ) where
 
-import qualified App.VPSScan.Scan.RunIPR as RunIPR
 import Control.Carrier.Diagnostics
 import Prologue
 import Text.URI (URI)
 import Network.HTTP.Req
 import Data.Text.Prettyprint.Doc (viaShow)
 
-data FossaOpts = FossaOpts
-  {fossaUrl :: URI, fossaApiKey :: Text}
-  deriving (Generic)
+newtype FilterExpressions = FilterExpressions String
+
+instance Show FilterExpressions where
+  show (FilterExpressions x) = x :: String
```

Comment:
rather than using `Show`, the usual pattern is to add a record field:
```hs
newtype FilterExpressions = FilterExpressions { unFilterExpressions :: Text }
```

you can use `unFilterExpressions` to extract the underlying `Text` (and kill this `Show` instance, though `deriving Show` could still be useful)